src/card/view/CardView.tsx

```diff
@@ -137,6 +140,11 @@ const NeoCardView = ({
     if (!settingsOpen) {
       setLastRunTimestamp(Date.now());
     }
+
+    // Resets the report with save button
+    if (enableSaveButtonForIds.map((report) => report.id).includes(id)) {
+      setActive(false);
+    }
   }, [JSON.stringify(localParameters)]);
```

Comment:
Why is it JSON.stringify in react hook and not just localParameters?

Reply:
I think localParameters is what we define. It can be nested as well. I am guessing that to detect, any changes in the localParameters will trigger this useEffect. I am not 100% sure.